### ISSUE TITLE
Hyprlock Shape size in percent acting weird

### ISSUE THREAD

**Issue by LJLP** [hyprwm/hyprlock#580]
### Regression?

No

### Hyprlock Info and Version

<Enter your hyprlock version here>Hyprlock version 0.5.0
<details> <summary>Hyprlock config</summary>

```sh
shape {
    monitor = DP-1
    size = 100%, 100%
    color = rgba(255, 255, 255, 1)
    position = 0, 0
    halign = center
    valign = center
    zindex = 2
}
```

</details>

### Compositor Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland 0.45.2 built from branch  at commit 12f9a0d0b93f691d4d9923716557154d74777b0a  ([gha] Nix: update inputs).
Date: Tue Nov 19 21:47:18 2024
Tag: v0.45.2, commits: 5451
built against aquamarine 0.5.0


flags set:
debug


System Information:
System name: Linux
Node name: archstation
Release: 6.12.1-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Fri, 22 Nov 2024 16:04:27 +0000


GPU information: 
09:00.0 VGA compatible controller [0300]: NVIDIA Corporation GA102 [GeForce RTX 3090 Ti] [10de:2203] (rev a1) (prog-if 00 [VGA controller])
NVRM version: NVIDIA UNIX x86_64 Kernel Module  565.57.01  Thu Oct 10 12:29:05 UTC 2024


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:
```

</details>

### Description

When adding a shape with the `size = 100%, 100%` to the lock screen it only creates a small square instead of growing to the size of the entire screen.

### How to reproduce

1. Add a `shape` to the Hyprlock config
2. `size = 100%, 100%`

### Crash reports, logs, images, videos

![shape_item_not_working-percent](https://github.com/user-attachments/assets/c6bffad6-1641-4a1c-bcd8-69a885e4f1d5)

**Comment by PaideiaDilemma** [comment]
That feature is not available in 0.5.0 yet

**Comment by LJLP** [comment]
Thank you for the help.
I switched to the git version of the package and it indeed works there.

I'd recommend the `hyprlock --version` command also report that the git version is installed.
It still reports `Hyprlock version 0.5.0` after switching to the git version.
